### DISCUSSION TITLE
website to search for packages

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [terrapkg/packages#2707] [Ideas]
I know that repology exists, but it is not the most beautiful and inviting, I think it would be interesting to have a specific website of yours

some example sites:
https://packages.vanillaos.org/
https://www.nixhub.io/

**Comment by madonuko** [comment]
We had plans for that but we don't have the manpower to do that.

**Reply by lleyton** [reply]
It's on my agenda once Ultramarine 41 is shipped :p
